core/spec/models/spree/stock_item_spec.rb

```diff
@@ -246,14 +246,6 @@
           end.not_to change { subject.variant.reload.updated_at }
         end
       end
-
-      context 'when a new stock location is added' do
-        it 'touches its variant' do
```

Comment:
Not sure if we need variant to be touched when creating a new stock location?

Reply:
@aplegatt we can run touch for all Variants at once via single SQL query:

```
Spree::Variant.touch_all
```

https://apidock.com/rails/ActiveRecord/Relation/touch_all

Reply:
And yes we need it to invalidate cache